lib/opensusebasetest.pm

```diff
@@ -47,6 +47,11 @@ sub save_and_upload_systemd_unit_log {
     $self->save_and_upload_log("journalctl --no-pager -u $unit", "journal_$unit.log");
 }
 
+sub detect_bsc_1063638 {
```

Comment:
why you decide to put it into opensusebasetest ? voting to move it to a single file where it is used

Reply:
I found that in this specific case one line function really confusing. All what you can get from function name is that some bug will be detected , no clue what this bug about and no clue how exactly it will be detected.

Reply:
It's generic issue which we see on all our distributions, so I believe it belongs here, as we have problem detection. I haven't put more comments as there is a bug for details, but I will add some, not a problem.

Reply:
@asmorodskyi if you have "no clue" about this bug I recommend you to read it, it's really worth to now – and you can see me being dizzed a lot :D

Reply:
@okurz this is not the point, of course I have read it but it does not mean that I should remember it's ID by heart anyway, right ? 
@rwx788   does not matter how generic issue is, I think that rule of thumb that files like utils.pm and opensusebase contains functions which used **at least** in two other files. And motivation "it might be used in more places" I call over engineering

Reply:
@asmorodskyi I see misunderstanding here. Now I got your point and it's a nice catch, because I initially added this detection to 2 places, and one of them is now missing.